moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1906,22 +1906,29 @@ double RobotState::computeCartesianPath(const JointModelGroup* group, std::vecto
 
   Eigen::Quaterniond start_quaternion(start_pose.rotation());
   Eigen::Quaterniond target_quaternion(rotated_target.rotation());
-  double distance = start_quaternion.dot(target_quaternion);
-  if (distance < 0)  // need to bring quaternions to same half sphere?
+
+  if (max_step.translation <= 0.0 && max_step.rotation <= 0.0)
```

Comment:
add comment

Reply:
I feel like the logError message replace the need for an additional comment.

Reply:
sure